ClientApp/viking-vault/src/Common/UserIconAvatar.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+ 
+interface IUserIconAvatarProps {
+    pictureUri: string;
+    pictureStyle: string;
+    defaultPicture: string;
+}
+ 
+interface IUserIconAvatarState {
+    isImageOk: boolean;
+}
+ 
+class UserIconAvatar extends React.Component<IUserIconAvatarProps, IUserIconAvatarState> {
+    constructor(props: any) {
```

Comment:
This feels weird...considering the origin of this code...shhhh. But the constructor in here in rendundant :))